indra/literature/pubmed_client.py

```diff
@@ -4,7 +4,7 @@
 import logging
 import requests
 from time import sleep
-from typing import List
+from typing import List, Literal
```

Comment:
If you want to import `Literal` this way, we need to make explicit that the minimum version is now python 3.8. Otherwise you can do `from typing_extensions import Literal` which backports all of this to earlier versions

Reply:
Personally, I think it's fine to bump to minimum 3.8 since 3.7 is past end of life

Reply:
I'm not thrilled about a breaking change for the sake of a type annotation.

Reply:
I can remove it

Reply:
Resolved in https://github.com/sorgerlab/indra/pull/1416/commits/7877aa10d19c5be755a966cdb7f5c616b8904501